.github/workflows/deploy.nightly.devnet.yml

```diff
@@ -116,6 +118,20 @@ jobs:
               polygon-edge polybft-secrets --data-dir {{ hostvars[item].tags["Name"] }} --json --insecure > {{ hostvars[item].tags["Name"] }}.json
           {% endif %}
           {% endfor %}
+
+          {% if (enable_eip_1559) %}
+              apt update
+              curl -fsSL https://deb.nodesource.com/setup_18.x | bash -
+              apt-get install -y nodejs
+
+              pushd /opt/polygon-edge/
+              make compile-core-contracts
+              cp -r /opt/polygon-edge/core-contracts /var/lib/bootstrap/core-contracts/
+              popd
+
+              BURN_CONTRACT_ADDRESS=0x0000000000000000000000000000000000000000
+          {% endif %}
```

Comment:
Don't need this either if it turns out that the `EIP1559Burn` is not needed. Agree?